refmterr.json

```diff
@@ -7,8 +7,8 @@
     "@opam/re": "*",
     "@esy-ocaml/reason": "^3.0.0",
     "ocaml": " >= 4.2.0  < 4.8.0",
-    "@reason-native/chalk": "*",
```

Comment:
Hey doesn't this still rely on `Chalk`? I think that during development you're not catching the build error because during development we develop as a monorepo. Another good reason to have some CI tests setup and another good reason for us to make sure monorepo development more closely resembles what is actually released.

Reply:
Resolved!